CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Added `fs.copy.copy_file_if`, `fs.copy.copy_dir_if`, and `fs.copy.copy_fs_if`. 
   Closes [#458](https://github.com/PyFilesystem/pyfilesystem2/issues/458).
   
```

Comment:
Could you add an entry to the **Added** section to note the `FS.getmodified` method was introduced here? Also, please add the `FS.getmodified` method to the list in `docs/source/interface.rst`.